index.html

```diff
@@ -83,9 +96,10 @@ <h3 class="project-title">Tonic</h3>
         </div>
       </section>
       <!-- project 2 -->
-      <section class="grid-items other-projects">
+      <section class="grid-items other-projects fonts reverse">
         <img
           class="project-img"
+          id="project-2"
           src="images/projects/project-2.png"
           alt="Project-1"
```

Comment:
- [ ] Kindly make sure your project cards look like the Figma design as requested in the project requirements, by using the proportions in terms of width and height.

Your design:
![Screenshot from 2023-02-24 14-09-22](https://user-images.githubusercontent.com/97834160/221176114-ad557c19-71c8-4e6e-8ce8-818b12c2581d.png)

Figma:

![Screenshot from 2023-02-24 14-13-11](https://user-images.githubusercontent.com/97834160/221176393-7ad81a77-3e28-4a57-9e16-824ac8112cb7.png)